themes/src/main/resources/theme/keycloak/common/resources/node_modules/angular-translate/dist/angular-translate.js

```diff
@@ -2014,7 +2014,7 @@ function $translate($STORAGE_KEY, $windowProvider, $translateSanitizationProvide
               deferred.reject(applyNotFoundIndicators(_translationId));
```

Comment:
This is not really our code, not sure if makes sense to change this one if they could be reverted once the library is updated.

@ssilvert makes sense?

Reply:
Yes, this is just a copy of sources from elsewhere so we shouldn't modify them